apps/andi/lib/andi/migration/owner.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Andi.Migration.Owner do
```

Comment:
nice, I don't think we've used this pattern for past migrations but we should have